ktoml-core/src/commonTest/kotlin/com/akuleshov7/ktoml/parsers/SetLineNoTest.kt

```diff
@@ -0,0 +1,58 @@
+package com.akuleshov7.ktoml.parsers
+
+import com.akuleshov7.ktoml.Toml
+import com.akuleshov7.ktoml.tree.nodes.TomlKeyValuePrimitive
+import kotlin.test.Test
+import kotlin.test.assertContentEquals
+import kotlin.test.assertEquals
+
+class SetLineNoTest {
+    @Test
+    fun checkingLineNumbers() {
+        val string = """
+            
+            # comment 1
+            
+            [a] # comment 2
+            # comment 3
+             test = 1 # comment 4
+             
+             # ====
+             
+             [[a.b]] # comment 5 
+                test = 1
+                
+             mls = '''
+                  1
+                  2
+                  3
+             '''
+             
+             mla = [
+                 "a",
+                 "b",
+                 "c"
+                 ]
+        """.trimIndent()
+        val parsedToml = Toml.tomlParser.parseString(string)
+
+        parsedToml.prettyPrint(true)
+        assertEquals(
+            """
+                 | - TomlFile (rootNode)[line:0]
+                 |     - TomlTablePrimitive ([a])[line:4]
+                 |         - TomlKeyValuePrimitive (test=1)[line:6]
+                 |         - TomlArrayOfTables ([[a.b]])[line:10]
+                 |             - TomlArrayOfTablesElement (technical_node)[line:10]
+                 |                 - TomlKeyValuePrimitive (test=1)[line:11]
+                 |                 - TomlKeyValuePrimitive (mls='''      1
```

Comment:
that's a little bit weird, cause it looks like we forget to put a newline after multiline string delimiters

Reply:
@NightEule5 while we encode the string, I think it is necessary for the user to have a proper formatting of multiline strings. Now we have the following: 
```
a = """1
2
3"""
```

I think (even the TOML spec is confusing [https://toml.io/en/v1.0.0#string](here)) that we should add a newline there (it will be more convenient):
```
a = """
1
2
3
"""
```

Reply:
Yep, true

Reply:
And of course I have forgotten to trim spaces before the quotes `'''` while decoding 😢 
https://github.com/akuleshov7/ktoml/issues/221